include/picongpu/param/particle.param

```diff
@@ -46,42 +46,51 @@ namespace picongpu
          *  unit: none */
         constexpr float_X MIN_WEIGHTING = 10.0;
 
-        /** Number of maximum particles per cell during density profile evaluation.
+        /** Number of maximum macro-particles per cell during density profile evaluation.
          *
-         * Determines the weighting of a macro particle and with it, the number of
-         * particles "sampling" dynamics in phase space.
+         * Determines the weighting of a macro particle as well as the number of
+         * macro-particles which sample the evolution of the particle distribution
+         * function in phase space.
          */
         constexpr uint32_t TYPICAL_PARTICLES_PER_CELL = 2u;
 
         namespace manipulators
         {
-            /** Parameter for DriftParam
+            /** Define initial particle drift direction vector.
+             *  Needs to be normalized to 1!
```

Comment:
This comment is not needed. The direction is normalized automatically [here](https://github.com/ComputationalRadiationPhysics/picongpu/blob/f2cfee4166013760bb387e3cf04c4091eecb1fa9/include/picongpu/particles/manipulators/unary/Drift.hpp#L68).